### PR TITLE
Makes upgraded prostics worth getting - Re-balances robotic lims

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Makes the costly upgraded prostics that are not round start, worth getting and upgrading rather then just going to robotics to get a legitmently better lim

## Why It's Good For The Game

Makes upgraded prostics not worthless point/mat sinks for no reason

## Changelog
:cl:
balance: As all things are not
/:cl:

Nerfs Robotic lims to not be 5 harm 4 burn but 2 harm 1 burn
Makes upgraded lims 3 harm 2 burn

### CODE REVIEW

File: code/modules/surgery/bodyparts/robot_bodyparts.dm [hunk 1]
```diff
@@ -268,38 +268,38 @@
...
 	name = "reinforced surplus prosthetic right leg"
-	desc = "A skeletal, robotic limb. This one is reinforced to provide better protection."
+	desc = "A skeletal, robotic limb. This one is reinforced to provide better protection, and is made of stronger parts."
 	icon = 'icons/mob/augmentation/surplus_augments.dmi'
-	brute_reduction = 1
-	burn_reduction = 1
-	max_damage = 30
+	brute_reduction = 3
+	burn_reduction = 2
+	max_damage = 55
```
Comment (Trilbyspaceclone) [inline code/modules/surgery/bodyparts/robot_bodyparts.dm:302]: 60 is max damage for robotic lims so 5 less.

**Comment by Poojawa** [comment]
why are printed cyborg limbs dogshit compared to surplus limbs with extra bits welded to them

**Comment by Trilbyspaceclone** [comment]
@Poojawa One takes time and mats to make, one is just metal and round start.

**Review by silicons — APPROVED** [review]
eh ok.

**Pull request merged**